app/components/topicPicker.js

```diff
@@ -67,66 +80,34 @@ export default React.createClass({
     // TODO: break this into its component pieces
     // const {topicId}: string = this.context.router.getCurrentParams();
 
+    console.log(this.props);
+
     return (
       <div className='topic-picker topic-picker--left'>
         <ul className='topic-picker__nav'>
-          <li data-topic='1'><b>Topic 1</b></li>
-          <li data-topic='2'><b>Topic 2</b></li>
-          <li data-topic='3'><b>Topic 3</b></li>
-          <li data-topic='4'><b>Topic 4</b></li>
+          { this.props.topicsTmp.map(topic => {
```

Comment:
Looking good, gents!

On Mon, Feb 15, 2016 at 10:59 AM, Tyler Burton <notifications@github.com>
wrote:

> In app/components/topicPicker.js
> <https://github.com/Goodly/annotator-content-analysis/pull/19#discussion_r52933327>
> :
>
> >      return (
> >        <div className='topic-picker topic-picker--left'>
> >          <ul className='topic-picker__nav'>
> > -          <li data-topic='1'><b>Topic 1</b></li>
> > -          <li data-topic='2'><b>Topic 2</b></li>
> > -          <li data-topic='3'><b>Topic 3</b></li>
> > -          <li data-topic='4'><b>Topic 4</b></li>
> > +          { this.props.topicsTmp.map(topic => {
>
> cool. i've got a branch out that has starts to use the actions / store
> stuff for this so if we get these two merged, i can then merge master into
> my branch, resolve any conflicts and we'll have a good pattern for going
> forward.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Goodly/annotator-content-analysis/pull/19/files#r52933327>
> .
>



-- 

*Nick Adams, Ph.D.Data Science Fellow, *

*Berkeley Institute for Data Science*
*University of California, Berkeley*